packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -428,9 +493,22 @@ class AuroraPostgresClusterEngine extends ClusterEngineBase {
           majorVersion: version.auroraPostgresMajorVersion,
         }
         : undefined,
+      features: version ? version._features : { s3Import: 's3Import', s3Export: 's3Export' },
```

Comment:
I don't like the fact the names are repeated here again... how about this?

```ts
  export class AuroraPostgresEngineVersion {
    // ...
    public readonly _features?: AuroraPostgresEngineFeatures;
    // ...
  }
```

And then you can do:

```suggestion
      features: version
       ? {
         s3Import: version._features.s3Import ? this.S3_IMPORT_FEATURE_NAME : undefined,
         s3Export: version._features.s3Export ? this.S3_EXPORT_FEATURE_NAME : undefined,
       }
       : {  s3Import: this.S3_IMPORT_FEATURE_NAME, s3Export: this.S3_EXPORT_FEATURE_NAME },
```

And we add the private static constants `S3_IMPORT_FEATURE_NAME` and `S3_EXPORT_FEATURE_NAME` to this class? Something like that ;p.